lib/popcorn.ex

```diff
@@ -300,4 +300,19 @@ defmodule Popcorn do
 
     gather_app_specs(deps, Map.merge(specs, new_specs))
   end
+
+  defp cp_gzip(paths, out_dir, handle_missing_file \\ fn _path -> :ok end) do
```

Comment:
Why we need that?

Reply:
Not sure what you mean, it was convenient to have a function that copies and gzips what it copied

Reply:
I mean why we need to gzip copied things instead of relying on gzip compression of webserver. Other user artifacts  aren't compressed.

Reply:
Web server doesn't compress anything, you need to provide compressed versions of files. Phoenix has a task for
it AFAIK.